src/controllers/workspace.controller.js

```diff
@@ -0,0 +1,23 @@
+const Workspace = require('../models/workspace.models');
+const {response_201, response_500} = require('../utils/responseCodes.utils')
+
+exports.createWorkspace = async (req, res) => {
+    const workspace = new Workspace({
```

Comment:
@criticic Just make sure that these details are provided or not. If not send error response
```suggestion
    const {name, description, workspaceId, userId} = req.body;
    if(!name || description || workspaceId || userId) // send error response
    // do as usuall
```

Reply:
again, the in the model, its checked(and it has appropriate error messages as well) if the required parameters are present, so on .save it will throw an error, which I have hadled